src/kv/test/kv_serialisation.cpp

```diff
@@ -276,13 +273,13 @@ TEST_CASE(
     view->put(k, v1);
     view->put(k2, v2);
 
-    auto [success, reqid, serialised] = tx.commit_reserved();
+    auto [success, reqid, buffer] = tx.commit_reserved();
     REQUIRE(success == kv::CommitSuccess::OK);
     kv_store.compact(view->end_order());
 
-    REQUIRE(
-      kv_store2.deserialise(serialised.replicated) ==
-      kv::DeserialiseSuccess::PASS);
+    std::vector<uint8_t> data_vec(
```

Comment:
In these tests creating the vector from the buffer to be passed into `deserialise` as it will accept a vector in normal operation